internal/provisioner/model/model.go

```diff
@@ -343,6 +347,8 @@ type EnvoyNetworkPublishing struct {
 	ServiceAnnotations map[string]string
 }
 
+//
+
```

Comment:
nit: can rm this empty comment/whitespace.

Reply:
done